package.json

```diff
@@ -4,7 +4,7 @@
   "homepage": "https://www.mozilla.org/en-GB/firefox/developer",
   "license": "Mozilla Public License",
   "name": "zsh-firefox-dev",
-  "version": "98.0b3",
+  "version": "rolling",
```

Comment:
Let's try it not sure.  I will test it straight away.

Need to set docker to test it auto for next time.